modules/claims_api/lib/claims_api/v2/disability_compensation_pdf_mapper.rb

```diff
@@ -453,9 +453,11 @@ def treatment_centers
 
       def get_treatments
         @auto_claim['treatments'].map do |tx|
-          center = "#{tx['center']['name']}, #{tx.dig('center', 'city')}, #{tx.dig('center', 'state')}"
+          if tx['center'].present?
+            center = "#{tx.dig('center', 'name')}, #{tx.dig('center', 'city')}, #{tx.dig('center', 'state')}"
+          end
           name = tx['treatedDisabilityNames'].join(', ')
```

Comment:
I know this isn't your change but I"m able to break this when testing the center changes. if this is nil, it throws an error. 
```
(byebug) tx['treatedDisabilityNames'].join(', ')
*** NoMethodError Exception: undefined method `join' for nil:NilClass
```

trying to confirm if these are conditionally required, but this made it all the way to the mapper either way.. :concerned-face:

Reply:
ya f it made it that far then there is no validation catching it.  Probably easy enough to include that update in this ticket since it is right there and then we do nto need to spin anything else up.  I will look at it.

Reply:
also, are we calling`.join(', ')` on an array of strings?

Reply:
Yes, looks like the goal there is changing `["Hearing Loss", "Traumatic Brain Injury"]` to `"Hearing Loss, Traumatic Brain Injury"`.